src/app/ErrorView/ErrorView.tsx

```diff
@@ -44,15 +44,31 @@ export interface ErrorViewProps {
 }
 
 export const ErrorView: React.FunctionComponent<ErrorViewProps> = (props) => {
```

Comment:
Related to the previous comment, I don't think I like the idea of implementing this as a specialization of the `ErrorView` - this precludes us from presenting this information to the user unless the failure occurs specifically when they are looking at a view where replacing the content with an `ErrorView` makes sense. I think a message dialog as called out in the original issue would be more versatile, since it can be presented in any scenario, and I think it also provides a better hint to the user that something is wrong but their action is required to (hopefully) fix it.

Reply:
So it would be similar to the JMX authentication modal? And I am guessing it would just present the error and the button to go to the security tab?

Reply:
Yes, pretty much.